hostfactory/1.1/providerplugins/azurecc/scripts/src/cyclecloud_provider.py

```diff
@@ -744,13 +753,15 @@ def _create_status(self, input_json, json_writer=None):
                     "name": hostname or "",
                     "status": machine_status,
                     "result": machine_result,
-                    "machineId": node.get("NodeId") or "",
+                    "machineId": self.cluster.get_node_id(node) or "",
                     # launchTime is manditory in Symphony
```

Comment:
Ah, it is used here - the large diffs didn't expand. I would just use node.delayed_node_id.node_id - it is a bit funky, and I should probably change it.

Reply:
it is used in 6 places in this file, initially I had used node.delayed_node_id.node_id but then as it was repetitive created a function :) you suggest create function in cyclecloud_provider.py?